File: employee.py
class Employee():
    c = 0
    ids_list = []

    def __init__(self, uid, name, gender, jobhour, amount):
        self.uid = uid
        self.name = name
        self.gender = gender
        self.jobhour = jobhour
        self.amount = amount
        Employee.c += 1
        self.ids_list.append([uid, name, gender, jobhour, amount])
        self.ids_list = sorted(self.ids_list, key=lambda x: x[3])
        self.ids_list = sorted(self.ids_list, key=lambda x: x[2])
        
        
    def set_uid(self, uid):
        self.uid = uid    
    def set_name(self, name):
        self.name = name
    def set_gender(self, gender):
        self.gender = gender
    def set_jobhour (self, jobhour):
        self.jobhour = jobhour
    def set_amount (self, amount):
        self.amount = amount
        
        
    def get_uid(self):
        return self.uid
    def get_name(self):
        return self.name
    def get_gender(self):
        return self.gender
    def get_jobhour(self):
        return self.jobhour
    def get_amount (self):
        self.amount = amount
    def __str__(self):
        return "\nId: "+ self.uid +"\nName: " +self.name+"\nGender "+self.gender+"\nJob Hour "+self.jobhour +"\nAmount "+self.amount



n = input("How many employees?")
e = []


for i in range(int(n)):
    name = input("Enter employee name")
    uid = input("Enter the ID number")
    gender = input("Enter Gender M/F: ")
    
  
    while True:
            try:
                jobhour = int(input('Enter Job  hours more than 10: '))
                if jobhour < 10:
                    raise ValueError 
                break
            except ValueError:
                print("Invalid integer. The number must be more than 10.")
                
   
    if (gender == "M"):
            
        while True:
            try:
                amount = int(input('Enter amount betwen 100 to 150: '))
                if amount < 100 or amount > 150:
                    raise ValueError 
                break
            except ValueError:
                print("Invalid integer. The number must be in the range of 100-150.")
                
        

    if (gender == "F"):       
        while True:
            try:
                amount = int(input('Enter amount betwen 70 to 120: '))
                if amount < 70 or amount > 120:
                    raise ValueError 
                break
            except ValueError:
                print("Invalid integer. The number must be in the range of 70-120.")
                
    entry = Employee(name, uid, gender, jobhour, amount)
        
    print(entry.ids_list)
    
   

File: Employee.java
import java.util.Scanner; 
import java.util.*;  
import java.util.stream.*;  
import static java.util.Comparator.*;

public class Employee
{
	private String id;
	private String name;
	private String gender;
	private int jobhour;
	private int amount;
	
	public Employee(){}
	
	public Employee(String id, String name,String gender, int jobhour, int amount){
	this.id = id;
	this.name = name;
	this.gender = gender;
	this.jobhour = jobhour;
	this.amount = amount;
	
	}
	
	public String getId() {
        return id;
    }
 
    public void setId(String id) {
        this.id = id;
    }
	
	public String getName() {
        return name;
    }
 
    public void setName(String name) {
        this.name = name;
    }
	
	
	public String getGender() {
        return gender;
    }
 
    public void setGender(String gender) {
        this.gender = gender;
    }
	
	public int getJobhour() {
        return jobhour;
    }
 
    public void setJobhour(int jobhour) {
        this.jobhour = jobhour;
    }
	
	public int getAmount() {
        return amount;
    }
 
    public void setAmount(int amount) {
        this.amount = amount;
    }
	
	
	public String toString(){
 
        return "[" + 
            this.getId() + 
            " : " + 
            this.getName() + 
            " : " + 
			this.getGender() + 
            " : " + 
			this.getJobhour() + 
            " : " + 
            this.getAmount() + 
            "]";
    }
	

    public static void main(String[] args) {
        System.out.println("Welcome to Online IDE!! Happy Coding :)");
    
	Employee em = new Employee();
	
	Scanner in = new Scanner(System.in);
	
	System.out.println("How Many Employee ? :");
	int n = in.nextInt();
	
	List<List<String>> list = new ArrayList<>();
	//List<String> list = new ArrayList<String>();
	
	for( int i =0; i < n;i++)
	{
		List<String> lis = new ArrayList<String>();
	
		System.out.println("Enter Employee  Id :");
		int jid = in.nextInt();
		lis.add(String.valueOf(jid));
		
		
		System.out.println("Enter Employee  Name :");
		String jname = in.next();
		lis.add(jname);
		
		
		System.out.println("Enter Employee  Gender M/F :");
		String jgender = in.next();
		lis.add(jgender);
		
		Boolean bFlag = true;
		
		while(bFlag) {
			try{
				System.out.println("Enter Employee  Job Hour more than 10! :");
				int jjobhour = in.nextInt();
				if (jjobhour > 10){
					lis.add(String.valueOf(jjobhour));

				bFlag = false;
				}
			}
			catch (InputMismatchException ex){
				System.out.print("You must input Job Hour more than 10!");
				int jjobhour = in.nextInt();
				bFlag = true;
			}
		}
		
				
		Boolean bFlag1 = true;
		
		if(jgender.equals("M")){
		while(bFlag1) {
			try{
				System.out.println("Enter Employee  Amount betwen 100 to 150!:");
				int jamount = in.nextInt();
				if (jamount >= 100 && jamount<= 150){
					lis.add(String.valueOf(jamount));
				bFlag1 = false;
				}
			}
			catch (InputMismatchException ex){
				System.out.print("You must input Amout betwen 100 to 150!");
				int jamount = in.nextInt();
				bFlag1 = true;
			}
		}
		}
		
		Boolean bFlag2 = true;
		if(jgender.equals("F")){
		
		while(bFlag2) {
			try{
				System.out.println("Enter Employee  Amount betwen 70 to 120!:");
				int jamount = in.nextInt();
				if (jamount >= 70 && jamount<= 120){
					lis.add(String.valueOf(jamount));
				bFlag2 = false;
				}
			}
			catch (InputMismatchException ex){
				System.out.print("You must input Amout betwen 70 to 120!");
				int jamount = in.nextInt();
				bFlag2 = true;
			}
		}
		}
		
	
		list.add(lis);
		//list.add(lis.toString());
	}


	System.out.println("Printing List: "+list);  
	
	
	list.forEach(System.out::println); 
	
}
}